src/controllers/stats.controller.ts

```diff
@@ -221,3 +223,18 @@ statsController.get(
     }
   },
 );
+
+statsController.get('/market/chart', async (req, res) => {
+  try {
+    const { period } = validateMarketChartsSchema.validateSync({
+      ...req.query,
```

Comment:
NB: no need to destructure req.query

Reply:
I just fixed. 
https://github.com/pastelnetwork/pastel-explorer-backend/commit/121e184e7f1413ec290f48833f9943831c101f4e